dtfs-central-api/src/helpers/convert-timestamp-to-date.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Converts the supplied timestamp to a date object
+ * after checking whether or not the timestamp is
+ * in seconds or milliseconds
+ * @param timestamp- The timestamp to convert
+ * @returns A date object
+ * @throws {Error} If the supplied timestamp cannot be converted to a unix timestamp
+ */
+export const convertTimestampToDate = (timestamp: string | number | Date): Date => {
+  if (!(timestamp instanceof Date) && !/^\d+$/.test(timestamp.toString())) {
+    throw new Error(`Expected timestamp to be either a date object, number or string of digits (received '${timestamp}')`);
+  }
+
+  const timestampAsNumber = Number(timestamp);
+  if (timestampAsNumber.toString().length === 10) {
+    return new Date(timestampAsNumber * 1000); // convert from seconds to milliseconds
+  }
+  return new Date(timestampAsNumber);
```

Comment:
Happy to make the one-line helper functions

Definitely should return early if the input is already a `Date`

I didn't think about the fact that the timestamp not being 10 digits doesn't mean it will be 13 - I'll make it so the function  throws if the timestamp does not match a format we expect